internal/pkg/local/load.go

```diff
@@ -29,7 +30,7 @@ func (m *Manager) loadObject(manifest model.ObjectManifest, object model.Object)
 
 	// Validate, if all files loaded without error
 	if errors.Len() == 0 {
-		if err := validator.Validate(object); err != nil {
+		if err := validator.Validate(ctx, object); err != nil {
```

Comment:
Pri template nastavujeme do `context`, ze sa ma ignorovat validator `required_in_project`.
Je to potrebne este upravit na viacerych miestach, ... preto som parameter `ctx` pridal priamo do `Validate`.